test/unit/parser/parser_black.h

```diff
@@ -259,8 +259,6 @@ class Cvc4ParserTest : public CxxTest::TestSuite, public ParserBlack {
     tryGoodInput("a : INT = 5; a: INT;"); // decl after define, compatible
     tryGoodInput("a : TYPE; a : INT;"); // ok, sort and variable symbol spaces distinct
     tryGoodInput("a : TYPE; a : INT; b : a;"); // ok except a is both INT and sort `a'
-    //tryGoodInput("a : [0..0]; b : [-5..5]; c : [-1..1]; d : [ _ .._];"); // subranges
-    tryGoodInput("a : [ _..1]; b : [_.. 0]; c :[_..-1];");
```

Comment:
Why did you remove those two lines? You don't need to explain it in the code. I just want to know.

Reply:
It was a comedy of errors that this worked previously :)

Essentially, this is a testing a feature (subrange types) that was supported in cvc3 but we never added support for in cvc4.  Moreover, the cvc parser incorrectly did not issue official define-fun commands until I fixed this a few PRs ago. Once this was fixed, this still did not trigger an issue since the parser was using the old API and did not complain about making variables of null types.  Now, we call the new API, which correctly recognized that this is not a supported input.